_CoqProject

```diff
@@ -120,3 +120,11 @@ theories/unavoidability.v
 theories/reducibility.v
 theories/combinatorial4ct.v
 theories/fourcolor.v
+
+-R theories fourcolor
```

Comment:
@ybertot I think this has gotten duplicated from the top of the file. The general convention in nearly all Coq-community projects is to put `-R`/`-Q` and warnings at the top of `_CoqProject`.